examples/integration_test/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
+
+services:
+  frontend:
+    image: "frontend"
+    profiles: [ "*" ]
+    platform: linux/amd64
```

Comment:
This Github runner can only create amd64 images, because qemu is not configured.